analysis_driver/dataset.py

```diff
@@ -116,15 +116,16 @@ def soft_terminate(self):
 
     def _terminate(self, signal_id):
         pid = self.most_recent_proc.get('pid')
-        self.debug('Attempting to terminate pid %s for %s %s with signal %s', pid, self.type, self.name, signal_id)
+        self.info('Attempting to terminate pid %s for %s %s with signal %s', pid, self.type, self.name, signal_id)
         if not pid or not self._pid_valid(pid):
-            self.debug('Attempted to terminate invalid pid %s with signal %s', pid, signal_id)
+            self.warning('Termination unsuccessful')
             return
 
         os.kill(pid, signal_id)
         while self._pid_running(pid):
             sleep(1)
-        self.info('Terminated pid %s for %s %s with signal %s', pid, self.type, self.name, signal_id)
+
+        self.info('Termination successful')
```

Comment:
None of these comment are ever seen because at the point of calling terminate we have not setup the file logging and I don't think stdout is setup either

Reply:
Have added a stdout logging handler to the manual commands (reset, skip, stop, soft stop, etc.)